test/utils/integration-tests.sol

```diff
@@ -45,9 +45,12 @@ import {
 
 uint256 constant MAINNET_CHAIN_ID = 1;
 uint256 constant HOLESKY_CHAIN_ID = 17000;
+uint256 constant HOODI_CHAIN_ID = 560048;
 
-uint256 constant DEFAULT_HOLESKY_FORK_BLOCK_NUMBER = 3209735;
 uint256 constant DEFAULT_MAINNET_FORK_BLOCK_NUMBER = 20218312;
+uint256 constant DEFAULT_HOLESKY_FORK_BLOCK_NUMBER = 3209735;
+uint256 constant DEFAULT_HOODI_FORK_BLOCK_NUMBER = 727;
```

Comment:
I propose to take a "fresher" block number. It's already 200k+ blocks

Reply:
Also, some parts of the protocol were deployed after block 727, for instance, wstETH at block 733 https://hoodi.etherscan.io/tx/0x10befa6d959ebdc7e0bda972214d5ac60d063f6178563c0cc3eb4e2a22cadd23